cmd/influxd/config.go

```diff
@@ -40,6 +40,10 @@ type Config struct {
 	Version           string `toml:"-"`
 	InfluxDBVersion   string `toml:"-"`
 
+	Authentication struct {
```

Comment:
Not technically no, but I think the code reads better. Otherwise we'll have a flag like:
```
auth-enabled
```
I guess it's not *that* bad.